skrf/calibration/calibration.py

```diff
@@ -3163,20 +3158,20 @@ def solve_A(B1, B2, CoA1, CoA2):
                     raise ValueError('Only one of c0 or z0_line can be given.')
                 try:
                     c0m = self.c0[m]
-                except TypeError:
+                except (TypeError, IndexError):
```

Comment:
Could this also be changed to use `isscalar` instead of `try ... else`? Either at `__init__` set it to array or `None` or check it here.

Reply:
I decided to convert scalar values to a list in the `__init__`. It takes a bit extra memory, but I think it is rare for the frequency to be so large this causes problems.

Note that the `z0_line`, etc. are public attributes so this slightly changes the public behavior of the TRL.